source/common/stats/stat_merger.h.save

```diff
@@ -0,0 +1,49 @@
+#pragma once
```

Comment:
I think you didn't mean to commit this file?

Reply:
Whoa, yeah.... thanks for noticing.